hypervisor/include/common/hypercall.h

```diff
@@ -51,6 +51,20 @@ int32_t hcall_sos_offline_cpu(struct acrn_vm *vm, uint64_t lapicid);
  */
 int32_t hcall_get_api_version(struct acrn_vm *vm, uint64_t param);
 
+
+/**
+ * @brief Get basic platform information
+ *
+ * The function return basic platform configurations information.
+ *
+ * @param vm Pointer to VM data structure
+ * @param param GPA pointer to struct hc_platform_info
+ *
+ * @pre Pointer vm shall point to SOS_VM
+ * @return 0 on success, non-zero on error.
```

Comment:
Same comment as above regarding the use of `@retval` to list out all possible return values